samples/tab-personal-sso-quickstart/ts/README.md

```diff
@@ -130,6 +124,13 @@ Tabs are Teams-aware webpages embedded in Microsoft Teams. A channel/group tab d
    - From the lower left corner, choose Upload a custom App
    - Go to your project directory, the ./AppPackage folder, select the zip folder, and choose Open.
    - Select Add in the pop-up dialog box. Your app is uploaded to Teams.
+## Build and Run
```

Comment:
Why are we adding it after Manifest setup?

Reply:
because first will setup then code then build and run